flict/flictlib/format/common.py

```diff
@@ -25,3 +25,26 @@
         'question': {'markdown': 'Q---'},
     },
 }
+
+def extract_compatibilities(verification):
+    """
+    this method assumes 1 package with 1 dependency
```

Comment:
Is there a reason for this quite arbitrary limitation?

Reply:
removed it